web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -113,15 +125,18 @@ const TeamReviews = ({ onBack, periodId }) => {
   const history = useHistory();
   const location = useLocation();
 
+  const [approvalMode, setApprovalMode] = useState(false);
   const [assignments, setAssignments] = useState([]);
   const [confirmOpen, setConfirmOpen] = useState(false);
   const [memberSelectorOpen, setMemberSelectorOpen] = useState(false);
+  const [nameQuery, setNameQuery] = useState('');
   const [query, setQuery] = useState({});
   const [reviewerSelectorOpen, setReviewerSelectorOpen] = useState(false);
   const [reviews, setReviews] = useState(null);
   const [selectedMember, setSelectedMember] = useState(null);
   const [selectedReviewers, setSelectedReviewers] = useState([]);
   const [selfReviews, setSelfReviews] = useState({});
+  const [showAll, setShowAll] = useState(false);
```

Comment:
This determines whether only direct reports are displayed (false) or all subordinates in the hierarchy (true).